src/arrays/referencing_array_and_its_elements/hard/q5/MainTest.java

```diff
@@ -0,0 +1,102 @@
+package arrays.referencing_array_and_its_elements.hard.q5;
+
+import global.BaseTest;
+import global.MethodTest;
+import global.exceptions.InvalidClauseException;
+import global.tools.TestOption;
+import global.variables.Clause;
+import global.variables.clauses.NewLine;
+import global.variables.clauses.PlaceHolder;
+import global.variables.clauses.StringLiteral;
+import global.variables.wrappers.Optional;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+import static global.utils.ArrayUtil.arrayToInput;
+import static global.utils.ArrayUtil.generateRandomArray;
+
+public class MainTest extends BaseTest {
+    // Java
+
+    public static int n = 10;
+    public static double pivot = 60.0;
+
+    public Clause[] testSentence() {
+        TestOption.isInputTest = true;
+        TestOption.defaultInput = arrayToInput(generateRandomArray(0.0, 100.0, n));
+        return new Clause[]{
+                new StringLiteral("Enter " + n + " double numbers:"),
+                new Optional(new StringLiteral(" ")),
+                new NewLine(),
+                new StringLiteral("The indices of students needing help are:"),
+                new Optional(new StringLiteral(" ")),
+                new NewLine(),
+                new PlaceHolder()
+        };
+    }
+
+    public void runMain() {
+        StudentsNeedHelp.main(new String[0]);
+    }
+
+    public static String answerFor(double[] a) {
+        String ans = "";
+        for (int i = 0; i < a.length; i++) {
+            if (a[i] < pivot) {
+                ans += i + " ";
+            }
+        }
+        return ans.trim();
+    }
+
+    static Stream<Arguments> mainInputProvider() {
+        return Stream.of(
+                Arguments.of(generateRandomArray(0.0, 100, n)),
+                Arguments.of(generateRandomArray(0.0, 100, n)),
+                Arguments.of(generateRandomArray(0.0, 100, n)),
+                Arguments.of(generateRandomArray(60.0, 100, n)),
+                Arguments.of(generateRandomArray(0.0, 60.0, n))
+        );
+    }
+
+    @ParameterizedTest
+    @MethodSource("mainInputProvider")
+    void printCorrectOutput(double[] input) throws InvalidClauseException {
+        TestOption.incorrectStructureErrorMessage = "Your main method does not correctly print the indices of students needing help.";
```

Comment:
Should say the program does not print the indices because the main method should not print the indices